integration/indexes_compat_test.go

```diff
@@ -342,17 +350,19 @@ func TestDropIndexesCompat(tt *testing.T) {
 			},
 			dropIndexName: "v_-1",
 		},
-		"AsteriskWithDropOne": {
-			toCreate: []mongo.IndexModel{
-				{Keys: bson.D{{"v", -1}}},
-			},
-			dropIndexName: "*",
-			resultType:    emptyResult,
-		},
-		"NonExistent": {
-			dropIndexName: "nonexistent_1",
-			resultType:    emptyResult,
-		},
+		// We need to figure out why these test cases pass even if there is no implementation for dropIndexes.
+		// TODO https://github.com/FerretDB/FerretDB/issues/3287
+		//"AsteriskWithDropOne": {
```

Comment:
Ok, for this test case I figured out. It is checked by the driver itself. I will remove the test case as it rather tests how driver works, not how FerretDB works.